src/App.tsx

```diff
@@ -1,15 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer
+    .filter((movie) => movie.title.toLowerCase().includes(query.toLowerCase())
+      || movie.description.toLowerCase().includes(query.toLowerCase()));
```

Comment:
wo eta?
Mb like that?
Or do this code more readable by yourself

```suggestion
  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
      const preparedTitle = title.toLowerCase();
      const preparedDescription = description.toLowerCase();
      const preparedQuery = query.toLowerCase();
      
      return preparedTitle.includes(preparedQuery) || preparedDescription(preparedQuery);
    }
```